corehq/apps/fixtures/views.py

```diff
@@ -125,10 +126,7 @@ def update_tables(request, domain, data_type_id=None):
                 field_name = options['update']
             if is_identifier_invalid(field_name) and 'remove' not in method:
                 validation_errors.append(field_name)
-        validation_errors = [_(
-            "\"%s\" cannot include special characters, begin or end with a space, "
-            "or begin with \"xml\" or a number") % e for e in validation_errors
-        ]
+        validation_errors = [_(INVALID_NAME_ERROR) % repr(e) for e in validation_errors]
```

Comment:
I don't think `gettext` can translate variables.

Reply:
Hmm, I intended to use `gettext_noop` on the constant to address that, but looks like I forgot. 8ec476f9a4b874a5ca632d0e2e5e8b3e6d22774c

Reply:
cool, thank you